### PR TITLE
Add cw1 section and subkeys doc

### LINKED ISSUE #83: Document cw1-subkeys usage
Related to https://github.com/CosmWasm/testnets/issues/51

### PR DESCRIPTION
**Pull request by orkunkl**
Related to https://github.com/CosmWasm/testnets/issues/51
Resolves https://github.com/CosmWasm/docs2/issues/83

I kept it simple and language is funny. I can make it formal if requested.

### CODE REVIEW

File: cw-plus/cw20/spec.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+The specification is split into multiple sections, a contract may only
+implement some of this functionality, but must implement the base.
+
+::: tip
+This section contains CW20 spec implementation details.
+If you are around to play with the contract,
+skip this section and go to the [next page](./setup.md)
+:::
+
+## Base
```
Comment (ethanfrey) [inline cw-plus/cw20/spec.md:22]: This is cut-and-paste from the README, right?

That is fine (and nit to publicize this more), but we need to note to update this if I update the spec (add iterator to list all accounts for example)

File: cw-plus/cw1/subkeys.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+```ts
+const client = await useOptions(coralnetOptions).setup(PASSWORD);
+const factory = CW1(client);
+```
+
+Upload the code and the contract:
+
+```ts
+// contract is already uploaded on coralnet: codeId -> 12
+// const codeId = await factory.upload();
```
Comment (ethanfrey) [inline cw-plus/cw1/subkeys.md]: Maybe remove some of the comments (the one explaining codeId -> 12 is good, the others confusing)

File: cw-plus/cw1/subkeys.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+const { address } = await client.getAccount()
+// contract.contractAddress -> 'coral1267wq2zk22kt5juypdczw3k4wxhc4z47mug9fd'
```
Comment (ethanfrey) [inline cw-plus/cw1/subkeys.md]: Although printing this out and a comment on how to reconnect later (not making a new one) is a good advice

File: cw-plus/cw1/subkeys.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+Let's delete friends address from admins, you would not want him to
+run away with the funds. Remove his address from admins and freeze the contract.
+Freezing means admins cannot be modified afterwards.
+
+```ts
+contract.updateAdmins([address])
+contract.freeze()
+```
+
+Let's give some allowance to your friends account, so he can buy a lambo:
```
Comment (ethanfrey) [inline cw-plus/cw1/subkeys.md:71]: :laughing:

File: cw-plus/cw1/subkeys.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+contract.increaseAllowance(friendAddr, {denom: "ushell", amount: "90000"})
+contract.allowance(friendAddr)
+```
+
+Now test if he can execute the message. Open another terminal screen:
+
+```ts
+const friendClient = await useOptions(coralnetOptions).setup(PASSWORD, KEY_FILE);
+const factory = CW1(friendClient)
+const contract = factory.use('coral1267wq2zk22kt5juypdczw3k4wxhc4z47mug9fd')
```
Comment (ethanfrey) [inline cw-plus/cw1/subkeys.md:88]: Okay, yes, this is good... just comment above about "print out contract.contractAddress for later"

File: cw-plus/cw1/subkeys.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+```ts
+contract.updateAdmins([address])
+contract.freeze()
+```
+
+Let's give some allowance to your friends account, so he can buy a lambo:
+
+```ts
+contract.increaseAllowance(friendAddr, {denom: "ushell", amount: "90000"})
```
Comment (ethanfrey) [inline cw-plus/cw1/subkeys.md:79]: You also need to `sendTokens` to the contract, I don't see that section

File: cw-plus/cw20/advanced.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Advanced Usage
-order: 3
+order: 4
```
Comment (ethanfrey) [inline cw-plus/cw20/advanced.md]: Maybe this belongs in "Overview" or "General" as these tips are valid for all contracts

**Review by ethanfrey — APPROVED** [review]
Nice stuff and I like the style.

A few points to address before merging. Thanks for setting this up

File: cw-plus/cw20/spec.md (hunk 1 above)
Comment (orkunkl) [reply cw-plus/cw20/spec.md:22]: Then we make sure documentation pipeline is attentive.

File: cw-plus/cw20/advanced.md (hunk 7 above)
Comment (orkunkl) [reply cw-plus/cw20/advanced.md]: Nice idea.

**Pull request merged**